src/main/java/org/openelisglobal/notifications/rest/NotificationRestController.java

```diff
@@ -31,12 +31,11 @@
 
 @RequestMapping("/rest")
 @RestController
-public class NotificationRestController {
+public class NotificationRestController extends ControllerUtills {
```

Comment:
This is a Rest Controller oler so it  should  extend  the` BaseRestController` instead